src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -482,7 +479,7 @@ internal void CloseWindow()
         {
             CloseWasExplicit = true;
 
-            if (string.IsNullOrEmpty(hostAnalyticsInfo.HostName))
+            if (string.IsNullOrEmpty(DynamoModel.HostAnalyticsInfo.HostName))
```

Comment:
One last question, should we use `DynamoModel.HostAnalyticsInfo?.HostName` everywhere in case the property is null?